ci/docker_ci.sh

```diff
@@ -138,13 +137,25 @@ fi
 # Test the docker build in all cases, but use a local tag that we will overwrite before push in the
 # cases where we do push.
 for BUILD_TYPE in "${BUILD_TYPES[@]}"; do
-  build_images "${BUILD_TYPE}" "${DOCKER_IMAGE_PREFIX}${BUILD_TYPE}${IMAGE_POSTFIX}:${IMAGE_NAME}"
+    image_tag="${DOCKER_IMAGE_PREFIX}${BUILD_TYPE}${IMAGE_POSTFIX}:${IMAGE_NAME}"
+    build_images "${BUILD_TYPE}" "$image_tag"
+
+    if ! is_windows; then
+        if [[ "$BUILD_TYPE" == "" || "$BUILD_TYPE" == "-alpine" ]]; then
+            # verify_examples expects the base and alpine images, and for them to be named `-dev`
+            dev_image="envoyproxy/envoy${BUILD_TYPE}-dev:latest"
```

Comment:
```suggestion
            dev_image="${DOCKER_IMAGE_PREFIX}${BUILD_TYPE}-dev:latest"
```

Reply:
that wouldnt work - it needs specifically to be `envoyproxy/envoy-(alpine-)dev`

the images are consumed here (altho this can be removed with this PR - there will be no need to retag the images)

https://github.com/envoyproxy/envoy/blob/e330009658366a518c54ee747caca0a5dae6abe9/ci/do_ci.sh#L165

and for example here

https://github.com/envoyproxy/envoy/blob/e330009658366a518c54ee747caca0a5dae6abe9/examples/brotli/Dockerfile-brotli#L1

atm this is broken in setec repo and on version publishing - with this pr, and specifically with this hard-coded, these should be fixed